app/assets/javascripts/home/HomeInsights.test.js

```diff
@@ -13,6 +13,9 @@ function testProps(props = {}) {
   };
 }
 
+jest.mock('./CheckStudentsWithLowGrades');
+jest.mock('./CheckStudentsWithHighAbsences');
```

Comment:
After updating `setupTests` to fail on unhandled promise rejections, these were failing.  Mocking these components loses the propTypes checking, but then we can do that in the test code below.  Ideally we'd get all this automatically, but not sure how to do this right now.